src/app/commands/cmd_paste.cpp

```diff
@@ -30,12 +33,17 @@ PasteCommand::PasteCommand()
 
 bool PasteCommand::onEnabled(Context* ctx)
 {
-  return App::instance()->inputChain().canPaste(ctx);
+  return ctx->isUIAvailable() ?
+    App::instance()->inputChain().canPaste(ctx) :
+    ctx->clipboard()->isImageAvailable();
```

Comment:
Same case as in cut, probably we can implement a default input chain for batch processing.